addons/stock/models/stock_move.py

```diff
@@ -476,14 +475,12 @@ def _propagate_date_log_activity(self, move_orig):
         msg = _("The scheduled date has been automatically updated due to a delay on <a href='#' data-oe-model='%s' data-oe-id='%s'>%s</a>.") % (doc_orig[0]._name, doc_orig[0].id, doc_orig[0].name)
         # write the message on each document
         for doc in documents:
-            if doc.activity_ids.filtered(lambda a: a.automated and doc_orig[0].name in a.note):
+            last_message = doc.message_ids[:1]
+            # Avoids to write the exact same message multiple times.
+            if last_message and msg in last_message.body.replace('"', "'"):
```

Comment:
this seems pretty brittle.. isn't there a field you can use on mail message to know the message is a delay alert? i see there is `email_layout_xmlid`, can you see for what it is used/if there will be side effects to setting it ? you could also set "model" i suppose